app/javascript/components/enrollment/steps/ExposureInformation.js

```diff
@@ -530,7 +619,6 @@ const staticValidations = {
   healthcare_personnel_facility_name: yup.string().max(200, 'Max length exceeded, please limit to 200 characters.').nullable(),
   laboratory_personnel_facility_name: yup.string().max(200, 'Max length exceeded, please limit to 200 characters.').nullable(),
   was_in_health_care_facility_with_known_cases_facility_name: yup.string().max(200, 'Max length exceeded, please limit to 200 characters.').nullable(),
-  member_of_a_common_exposure_cohort_type: yup.string().max(200, 'Max length exceeded, please limit to 200 characters.').nullable(),
```

Comment:
We've removed this validation, but I think we need to validate the cohort name/description and cohort location.

Reply:
since these inputs are in a modal, the validation behaves differently and it prevents users from entering more than 200 characters: https://github.com/SaraAlert/SaraAlert/pull/1125/commits/237545888b608d51912da918ac3e3e6d0780ee28#diff-97fb78554e2c9392e787e0a8353b4bf0dafc28f73344f230aafc03b734078e3cR31